src/_locales/sw/messages.json

```diff
@@ -0,0 +1,500 @@
+  },
+  "optionEcMedium": {
+    "message": "Kati (15%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "0ea96235848be6f66788cad72bc0ea02"
+  },
+  "optionEcQuartile": {
+    "message": "Robo (25%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "344e853acdaef0ac95e161cdbc201036"
+  },
+  "optionEcHigh": {
+    "message": "Juu (30%)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction.",
+    "hash": "30035374d52e135473d464fe1726ad8c"
+  },
+  "optionErrorCorrectionDescr": {
+    "message": "Thamani hii inaonyesha ni kiasi gani cha nambari inaweza kukosa, ili iweze kusanibiwa.",
+    "description": "The description of the error correction option.",
+    "hash": "2bbbd8aafc044a910d26cbf04e560766"
+  },
+  "optionErrorCorrectionDescrLink": {
+    "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
+    "description": "The 'Learn more' link in the description of the error correction option.",
+    "hash": "530f6b1ee177dddd7d40571d98a64408"
+  },
+  "optionAutoGetSelectedText": {
+    "message": "Tumia kiotomati maandishi yaliyochaguliwa kwenye wavuti",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "21667c5a30f89561cea02351172b9c91"
+  },
+  "optionUseMonospaceFont": {
+    "message": "Tumia fonti ya monospace",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "69191273c436064be8c4f5d6a8f675f5"
+  },
+  "optionDebugMode": {
+    "message": "Washa hali ya kutatua",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "de6506af4973e0980d64f242fba11a19"
+  },
+  "optionDebugModeDescr": {
+    "message": "Hii ni muhimu tu kupata matokeo ya kina ya ukataji wa miti kwenye koni ya kuripoti kasoro.",
+    "description": "The description shown for the debug mode (optionDebugMode).",
+    "hash": "2aff10770796694827ec4253860e0771"
+  },
+  "translatorCredit": {
+    "message": "Ongeza hii imetafsiriwa kwa Kiingereza na $ TRANSLATORS $.",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": {
+        "content": "$1",
+        "example": "<a href=\"https://github.com/rugk/\">@rugk</a>"
+      }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/Paulmburu",
+    "description": "The link to the translator's GitHub profile."
+  },
+  "translatorUsername": {
+    "message": "Paulmburu",
+    "description": "The translator's GitHub username."
+  },
+  "contributorsThanks": {
+    "message": "Pia shukrani kwa $ WAKUWA WA $ $.",
```

Comment:
I guess you mistakenly translated the part inside of the `$$` signs too. Please do not do that. That is a placeholder, where other text is inserted.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).